module/zfs/spa.c

```diff
@@ -11074,6 +11194,12 @@ ZFS_MODULE_PARAM(zfs_livelist_condense, zfs_livelist_condense_, new_alloc, INT,
 	"Whether extra ALLOC blkptrs were added to a livelist entry while it "
 	"was being condensed");
 
+ZFS_MODULE_PARAM(zfs_spa, spa_, note_txg_time, UINT, ZMOD_RW,
+	"How often txg should be registred in database");
+
+ZFS_MODULE_PARAM(zfs_spa, spa_, flush_txg_time, UINT, ZMOD_RW,
+	"How often txg database should be flushed");
```

Comment:
Or maybe: "How frequently the TXG timestamps database should be flushed to disk (in seconds)"

Reply:
DOne.

Reply:
Actually wouldn't this be in minutes, since you currently aren't storing seconds resolution?

Reply:
No its a seconds, we do gethrestime_sec() and add the `spa_flush_txg_time`.
Current setting is `60 * 5` which should give 5 minutes. Or have I miss something?